### ISSUE TITLE
Store project names separately

### ISSUE THREAD

**Issue by elextr** [codebrainz/cleany#1]
Nice idea, I have used `geany -c projectdir` in the past for projects that needed configuration beyond that available in projects (I may have mentioned that projects should store all configuration ... repeatedly :).

The downside is that you can't easily swap from project to project when using `-c`, you have to start a new Geany/cleany by either knowing the path to the config or cd-ing to the project dir. 

If your script saved project names vs project dir in a known place then `cleany projectname` could look up (or create if not existing) the project directory and cd to it before running Geany with suitable arguments.

I assume using both -c and a project is for the benefit of the project plugins?

**Comment by codebrainz** [comment]
Yeah, you would fire up another instance/project. You only have to point `cleany` at the project directory and it should load up just fine.

**Comment by elextr** [comment]
Just had the thought, I'll make soft links in home to the project dirs, solved (unless you want to make cleany do that too :)
